src/views/accounts/AccountChangeSecret/index.vue

```diff
@@ -38,13 +38,22 @@ export default {
             name: 'ChangeSecretRecord',
             hidden: () => !this.$hasPerm('accounts.view_changesecretrecord'),
             component: () => import('@/views/accounts/AccountChangeSecret/ExecutionDetail/AccountChangeSecretRecord.vue')
+          },
+          {
+            title: this.$t('ChangeSecretAccount'),
+            name: 'ChangeSecretAccount',
+            hidden: () => !this.$hasPerm('accounts.view_changesecretexecution') || !this.ChangeSecretAfterSessionEnd,
+            component: () => import('@/views/accounts/AccountChangeSecret/AccountList.vue')
           }
         ]
       }
     }
   },
   computed: {
-    ...mapGetters(['hasValidLicense'])
+    ...mapGetters(['hasValidLicense']),
+    ChangeSecretAfterSessionEnd() {
+      return this.publicSettings?.CHANGE_SECRET_AFTER_SESSION_END
+    }
   },
   mounted() {
     this.$eventBus.$on('change-tab', this.handleChangeTab)
```

Comment:
The given code is incomplete and contains several issues such as missing import statements, incorrect props handling in the `export` sections, wrong method names used to retrieve permissions ('ChangeSecretAfterSessionEnd' instead of 'view_changesecretrecord'), and inconsistent use of template strings. Additionally, it seems there is an error with the `.hidden()` function call. Please review only what is provided here for more accurate feedback.

For specific details about which parts require changes/improvements or if anything else is unclear, I would need you to include further context or complete the snippet.